### ISSUE TITLE
Senza Template not working... a typo?

### ISSUE THREAD

**Issue by isnok** [zalando/stups2go#21]
Dear stups-Team,

I'm trying to deploy a gocd-server. It went all smooth until I want to use the given senza template.
That fails, roughly as follows:

```shell
$ senza create senza-go-server.yaml 1 [...] AppdynamicsApplication=gocd-server [...]
Generating Cloud Formation template.. EXCEPTION OCCURRED: Key 'Arguments.AppdynamicsAplication' not found: missing 'AppdynamicsAplication'
Traceback (most recent call last):
  File "/usr/local/bin/senza", line 11, in <module>
[...]
  File "/usr/local/lib/python3.4/dist-packages/pystache/renderengine.py", line 20, in context_get
    return stack.get(name)
  File "/usr/local/lib/python3.4/dist-packages/pystache/context.py", line 301, in get
    raise KeyNotFoundError(name, "missing %s" % repr(part))
pystache.context.KeyNotFoundError: Key 'Arguments.AppdynamicsAplication' not found: missing 'AppdynamicsAplication'
```

Application with only one "p" looks a bit suspicious to me.

**Comment by isnok** [comment]
Update: I left out the Parameter (since it's optional), and that worked without a flaw.

**Comment by whiskeysierra** [comment]
Fixed with #24
